overview.dd

```diff
@@ -876,15 +840,10 @@ $(SECTION3 $(LNAME2 resource, Resource Management),
 
     $(SECTION4 Automatic Memory Management,
 
-    $(P D memory allocation is fully garbage collected. Empirical experience
-    suggests that a lot of the complicated features of C++ are necessary
-    in order to manage memory deallocation. With garbage collection, the
-    language gets much simpler.
-    )
-
-    $(P Garbage collection eliminates the tedious, error prone memory
-    allocation
-    tracking code necessary in C and C++. This not only means much
+    $(P D memory allocation is fully garbage collected.
```

Comment:
This will raise red flags for people who think you can't allocate memory outside the garbage collector. Perhaps say "By default, D memory allocation.." but add a small sentence stating that manual memory management is still possible.